packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2905,6 +2905,13 @@ class _NotAnnounced extends Route<void> {
   // change has been announced.
 }
 
+/// Disposes navigation related singletons.
```

Comment:
Same comment here about documentation. Also, should this be a static methods on e.g. Navigator? It doesn't seem to belong in the global name space.